### PR TITLE
Alpdk horizontal shooting

### PR DESCRIPTION
**Pull request by alpdk**
Main idea, when player is moving he change his/her eyesight, but it stay the same if he/she stop moving.

### CODE REVIEW

File: Assets/Scripts/Characters/PlayerBehaviour.cs [hunk 1]
```diff
@@ -73,9 +84,7 @@ public class PlayerBehaviour : CharacterBehaviour<PlayerState>, IEdible {
     }
 
     private void Update() {
-        Vector2 mousePosition = mainCamera.ScreenToWorldPoint(Input.mousePosition);
-        Vector2 lookDirection = mousePosition - (Vector2)transform.position;
-        LookInDirection(lookDirection);
+        Vector2 lookDirection = GetEyeSightDirection();
```
Comment (InnocentusLime) [inline Assets/Scripts/Characters/PlayerBehaviour.cs:87]: Update the look direction in `Update`.

**Review by InnocentusLime — CHANGES_REQUESTED** [review]
More or less okay. Fix the notes please and we can merge it :)

File: Assets/Scripts/Characters/PlayerBehaviour.cs [hunk 2]
```diff
@@ -53,7 +56,15 @@ public class PlayerBehaviour : CharacterBehaviour<PlayerState>, IEdible {
 
     public override bool ShouldJump() => Input.GetKey(KeyCode.Space);
 
-    public override float GetWalkSpeed() => horizontalSpeed * Input.GetAxisRaw("Horizontal");
+    public override float GetWalkSpeed() {
+        float moveDirection = horizontalSpeed * Input.GetAxisRaw("Horizontal");
+
+        if (moveDirection != 0) {
+            LookInDirection(new Vector2(Mathf.Sign(moveDirection), 0));
+        }
```
[the remark this answers is not in the document]
Comment (alpdk) [reply Assets/Scripts/Characters/PlayerBehaviour.cs:64]: Ok, I agree with this opinion, and I will change it.

File: Assets/Scripts/Characters/PlayerBehaviour.cs (hunk 1 above)
Comment (alpdk) [reply Assets/Scripts/Characters/PlayerBehaviour.cs:87]: Ok.

File: Assets/Scripts/Characters/PlayerBehaviour.cs [hunk 3]
```diff
@@ -84,6 +76,12 @@ public class PlayerBehaviour : CharacterBehaviour<PlayerState>, IEdible {
     }
 
     private void Update() {
+        float walkSpeed = GetWalkSpeed();
+
+        if (walkSpeed != 0) {
+            LookInDirection(new Vector2(Mathf.Sign(walkSpeed), 0));
```
Comment (InnocentusLime) [inline Assets/Scripts/Characters/PlayerBehaviour.cs:82]: There is no need to do the `Mathf.Sign` stuff. The eyesight API normalises the vector it receives

**Review by InnocentusLime — APPROVED** [review]

**Comment by InnocentusLime** [comment]
Closes #62

**Pull request merged**